### DISCUSSION TITLE
Hide empty folders from viewers

### DISCUSSION THREAD

**Discussion by krzd7** [grafana/grafana#61131] [Dashboards, Panels & Visualizations]
It would be great having an option to hide empty folders from viewers. I'm not sure if there is a way, but I could not find one.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82898. Feel free to continue the discussion around this there. Thank you!
